index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+
+        <title>Intuit Accounts - Sign Up</title>
+
+        <link rel="stylesheet" href="css/styles.css">
+    </head>
+    <body>
+        <section>
+            <div>
+                <header>
+                    <div class="logo-container">
+                        <a href="#"><img src="images/intuit-logo.png"></a>
+                    </div>
+                    <div>
+                        <img src="images/ecosystem_logos_new.png">
```

Comment:
Hey guys!! just a suggestion you should use separated images or icons instead of one image